drivers/at86rf215/at86rf215.c

```diff
@@ -66,7 +66,7 @@ void at86rf215_reset_and_cfg(at86rf215_t *dev)
 
     /* set device address */
     luid_get_short((network_uint16_t *)&dev->netdev.short_addr);
-    luid_get_eui64((eui64_t *)&dev->netdev.long_addr);
+    luid_netdev_get_eui64(&dev->netdev.netdev, (eui64_t *)&dev->netdev.long_addr);
```

Comment:
Couldn't we add some `ieee802154_setup()` that would be called in the driver's setup function and would perform the netdev registration and obtain the address? Maybe more common stuff can be moved there in the future. (It wouldn't hurt to make this `static inline`, as we don't expect multiple netdevs of the some class on a single board.)